components/automate-cli/pkg/verifyserver/services/batchcheckservice/batchcheckservice.go

```diff
@@ -91,38 +94,44 @@ func (ss *BatchCheckService) getCheckInstance(check string) trigger.ICheck {
 		return ss.CheckTrigger.ExternalPostgresCheck
 	case constants.NFS_BACKUP_CONFIG:
 		return ss.CheckTrigger.NfsBackupConfigCheck
+	default:
+		return nil
 	}
-	return nil
 }
 
 func constructBatchCheckResponse(checkTriggerRespMap map[string][]models.CheckTriggerResponse,
 	hardwareDetails models.Hardware, checks []string) models.BatchCheckResponse {
-	//var result []models.BatchCheckResult
-	//var resultIndex = 0
-	//for k, v := range batchApisResultMap {
-	//	result[resultIndex].Ip = k
-	//	result[resultIndex].NodeType = getNodeTypeFromIp(k, hardwareDetails)
-	//	result[resultIndex].Tests = v
-	//	resultIndex = resultIndex + 1
-	//}
-	return models.BatchCheckResponse{
-		Status: "SUCCESS",
-		//Result: result,
+	ipMap := make(map[string][]models.CheckTriggerResponse)
+	for k, v := range checkTriggerRespMap {
+		for _, checkResp := range v {
+			checkIndex, _ := getIndexOfCheck(checks, k)
```

Comment:
can you please re-check this logic, and try to mock the responses from all 13 checks. I feel this may break

Reply:
Checked and updated the logic